src/Dev/SapphireTest.php

```diff
@@ -282,7 +282,9 @@ protected function setUp(): void
 
         // i18n needs to be set to the defaults or tests fail
         if (class_exists(i18n::class)) {
-            i18n::set_locale(i18n::config()->uninherited('default_locale'));
+            $defaultLocale = 'en_US';
+            i18n::config()->set('default_locale', $defaultLocale);
```

Comment:
Why are we changing the config here? We don't need to set the active locale, done we?

Reply:
`Requirements_Backend::add_i18n_javascript()` reads `i18n::config()->get('default_locale')` directly, it's what caused the unit test to fail in the first place